post_installation_configuration/ibmz-post-install.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: ASSEMBLY
+[id="post-install-configure-additional-devices-ibmz"]
+= Configuring additional devices in an {ibmzProductName} or LinuxONE environment
+include::_attributes/common-attributes.adoc[]
+:context: post-install-configure-additional-devices-ibmz
+
+toc::[]
+
+After installing {product-title}, you can configure additional devices for your cluster in an {ibmzProductName} or LinuxONE environment, which is installed with z/VM. The following devices can be configured:
+
+* Fibre Channel Protocol (FCP) host
+* FCP LUN
+* DASD
+* qeth
+
+You can configure devices by adding udev rules using the Machine Config Operator (MCO) or you can configure devices manually.
+
+[NOTE]
+====
+The procedures described here apply only to z/VM installations. If you have installed your cluster with {op-system-base} KVM on {ibmzProductName} or LinuxONE infrastructure, no additional configuration is needed inside the KVM guest after the devices were added to the KVM guests. However, both in z/VM and {op-system-base} KVM environments the next steps to configure the Local Storage Operator and Kubernetes NMState Operator need to be applied. 
+====
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../post_installation_configuration/machine-configuration-tasks.adoc#post-install-machine-configuration-tasks[Post-installation machine configuration tasks] 
+
+include::modules/ibmz-configure-devices-mco.adoc[leveloffset=+1]
+
+.Next steps
+
+* xref:../storage/persistent_storage/persistent-storage-local.adoc#persistent-storage-using-local-volume[Install and configure the Local Storage Operator]
+* xref:../networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc#k8s-nmstate-updating-node-network-config[Updating node network configuration]
+
+include::modules/ibmz-configure-devices-manually.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+See link:https://www.ibm.com/docs/en/linux-on-systems?topic=linuxonibm/com.ibm.linux.z.ludd/ludd_c_perscfg.html[Persistent device configuration] in IBM Documentation.
+
+[id="roce-network-cards"]
+== RoCE network Cards 
+RoCE (RDMA over Converged Ethernet) network cards do not need to be enabled and their interfaces can be configured with the Kubernetes NMState Operator whenever they are available in the node. For example, if they are attached in a z/VM environment or passed through in a {op-system-base} KVM environment. 
```

Comment:
"For example, if they are attached in a z/VM environment or passed through in a {op-system-base} KVM environment. " is a sentence fragment and needs revised